libvast/vast/json.hpp

```diff
@@ -101,69 +100,60 @@ class json : detail::totally_ordered<json> {
     return x.data_ < y.data_;
   }
 
+  template <class... Ts>
+  static json::array make_array(Ts&&... xs) {
+    return json::array{json{std::forward<Ts>(xs)}...};
+  }
+
 private:
   variant data_;
 };
 
 namespace detail {
 
-template <class, class = void>
-struct jsonize_helper : std::false_type {};
-
-template <>
-struct jsonize_helper<caf::none_t> {
-  using type = json::null;
-};
+template <class T,
+         int = std::is_convertible_v<T, json::number>
+               && !std::is_same_v<T, json::boolean>
+               ? 1
+               : (std::is_convertible_v<T, std::string> ? 2 : 0)>
+struct jsonize_helper;
```

Comment:
I find this formatting a bit more readable:
```
template <
  class T,
  int = std::is_convertible_v<T, json::number>
        && !std::is_same_v<T, json::boolean>
        ? 1
        : (std::is_convertible_v<T, std::string> ? 2 : 0)
>
```
But you can leave it as-is, just indent the line starting with `int` by one space.